Allow options to go directly to the first search result without opening the search engine

VSRonin opened the pull request.

VSRonin committed:
Allow using "I'm feeling lucky" feature on google and "Ducky" feature on DuckDuckGo to go directly to the first search result

VSRonin committed:
Fixed escaping of &

JKSH reviewed (changes requested):
Thanks! `btnI&q=` Doesn't seem to work for me though. According to https://productforums.google.com/forum/#!topic/websearch/pywUIBP6C20 it needs to be `btnI=I%27m+Feeling+Lucky&q=`

Also, please make these changes:
- Add parentheses to the names: "Google (I'm Feeling Lucky)" and "DuckDuckGo (Ducky)"
- Sort by alphabetical order (put the Ducky/Lucky entries under the normal DuckDuckGo/Google entries)
- Use tabs instead of space

VSRonin committed:
Fixed requests in codereview:

+ Add parentheses to the names: "Google (I'm Feeling Lucky)" and "DuckDuckGo (Ducky)"
+ Sort by alphabetical order (put the Ducky/Lucky entries under the normal DuckDuckGo/Google entries)
+ Use tabs instead of space
